addons/portal/controllers/mail.py

```diff
@@ -130,7 +130,8 @@ def portal_chatter_post(self, res_model, res_id, message, redirect=None, attachm
         attachment_tokens = [attachment_token for attachment_token in attachment_tokens.split(',') if attachment_token]
         self._portal_post_check_attachments(attachment_ids, attachment_tokens)
 
-        if message or attachment_ids:
+        # A rating value can be created w/o a feedback but not without a chatter post
+        if message or attachment_ids or kw.get('rating_value'):
```

Comment:
this condition should be in `portal_rating` to respect our module decomposition.